2. Web API now calls Microsoft Graph/README-incremental-instructions.md

```diff
@@ -158,12 +160,9 @@ Nothing more to do more here. All was done in the first part
 
 Clean the solution, rebuild the solution, and run it
 
-### Current limitations
+### Alternative architecture
```

Comment:
The limitation has gone now. the 3rd chapter proposes an interesting alternative architecture where client and service have the same client ID.